src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -36,7 +36,6 @@ const createNewStep = (context, step, args) => {
           name: 'pipeline-remoter-client',
           task,
           ...perSample && { 'sampleUuid.$': '$.sampleUuid' },
-          ...!perSample && { sampleUuid: 'all' },
```

Comment:
The default `SAMPLE_ID` if none is specified (i.e. the task is not sample-specific) should be no env variable at all (pipeline will convert that into an empty string, which is what R expects).